### PR TITLE
OFS-25: Added new model HistoryModel

### PR DESCRIPTION
**Pull request by sniedzielski**
This pull request delivers a whole new Model in models.py - HistoryModel

### CODE REVIEW

**Comment by sniedzielski** [comment]
OFS:25 https://openimis.atlassian.net/jira/software/c/projects/OFS/issues/?filter=allopenissues

**Review by delcroip — CHANGES_REQUESTED** [review]

File: core/models.py [hunk 1]
```diff
@@ -550,3 +552,57 @@ def mark_as_failed(self, error):
         MutationLog.objects.filter(id=self.id)\
             .update(status=MutationLog.ERROR, error=error)
         self.refresh_from_db()
+
+
+class HistoryModel(DirtyFieldsMixin, BaseVersionedModel):
```
[the remark this answers is not in the document]
Comment (sniedzielski) [reply core/models.py:557]: Ok, I'll change it.

[the remark this answers is not in the document]
Comment (sniedzielski) [reply core/models.py]: Ok. I've changed it.

File: core/models.py [hunk 2]
```diff
@@ -577,8 +577,6 @@ def save(self, *args, **kwargs):
            #save the new object
```
Comment (delcroip) [inline core/models.py:577]: Can you confirm that django will understand self.id without aliases like id = AliasField(db_column='uuid') ?

Comment (sniedzielski) [reply core/models.py:577]: Ok, I want to ask you question to be sure about "id" vs "uuid". You want to replace normal, standard "id" (integer value) into "uuid", don't you? For example there will be only uuid as a primary key and "id" should be the same as uuid. Is it exactly what you want? If yes, I'm gonna to fix it so as to have only uuid/id as a uuid. @dborowiecki mentioned to me earlier about this problem and we were wondering whether should be only "uuid" as "uuid" or "id/uuid" as a "uuid" or "id" as an integer.

**Review by delcroip — APPROVED** [review]

**Pull request merged**